lib/commands/plugin-test.sh

```diff
@@ -33,11 +35,32 @@ plugin_test_command() {
         fail_test "list-all did not return any version"
     fi
 
-    (asdf install $plugin_name ${versions[0]})
+    latest_version=${versions[0]}
```

Comment:
Yep, thanks, we were discussing  about the order and I think we are now settled :smile: